session.go

```diff
@@ -394,7 +394,8 @@ func (s *Session) reconnectDownedHosts(intv time.Duration) {
 			if gocqlDebug {
 				buf := bytes.NewBufferString("Session.ring:")
 				for _, h := range hosts {
-					buf.WriteString("[" + h.ConnectAddress().String() + ":" + h.State().String() + "]")
+					connAddr, _ := h.ConnectAddress()
```

Comment:
we shouldn't really ignore these errors, it's better to handle them than assuming that `connAddr` is always valid, just `continue` if unable to get `connAddr`